### ISSUE TITLE
Push Notifications / Email Digest to Keep People up on New Releases

### ISSUE THREAD

**Issue by maierb** [mage2tv/issues#30]
As you are seeding the website with content - one strategy we can pull from the ecommerce playbook is notifications. They are often thrown at us by sites where we are a random shopper... The difference with Mage2.tv is people should be able to opt in for notifications based on the following: (and only after subscribing?)

None of this is spam because: Users need to know of latest content, and users should get the most out of their subscription.
1. Browser push notifications upon video releases
2. Email Digest / weekly? / Monthly? 
3. Ability to "Notify me when this section becomes available" --> Very similar to out of stock notifications in Magento (for subscribed or unsubscribed since that may be part of why they haven't subscribed yet)

**Comment by Vinai** [comment]
Thanks for the input!
I'm currently sending out email notifications whenever a topic section is completed, so not every time I publish new content. The thought is to limit the amount of emails.
The site offers the option to subscribe to notifications when registering or through the account page.
I don't want to do browser push notifications because I really dislike them myself ;)

I like the idea of "Notify me when this section becomes available"! Need to kep that in mind...

**Comment by Vinai** [comment]
Related topic: https://github.com/mage2tv/issues/issues/28 (sort videos by date published).
I've already annotated the videos with the publishing date, I need to implement the sorting.

**Comment by johnexhut** [comment]
Yes, Push Notification is the best way to interact with your audience. As there is no contact detail require, just single click of subscription.
- No need to open a particular application or mail account, notification delivered as soon as user open the browser.
- It is an effective way of marketing and campaigning

**Comment by Vinai** [comment]
I personally really dislike push notifications and always disallow them, so I will have to come to terms with the idea before implementing them. For now I've completed the "sort by date" feature on the listing page, which allows users to get the relevant information if they want to.
